Sources/edge/Commands/Devices+default.swift

```diff
@@ -0,0 +1,16 @@
+#if !os(macOS) && !os(Linux)
+    import Foundation
+    import Logging
+
+    struct PlatformDeviceDiscovery: DeviceDiscovery {
+        func listUSBDevices(logger: Logger) {
+            print("Device listing is not supported on this platform")
```

Comment:
Should we print this to stderr instead of stdout?

Reply:
hm... I was torn. I think it's front and center for the user to see.

Reply:
I will change this to hardware specifically for the internals of the "device"